testing/test_sct_dmri_moco.py

```diff
@@ -44,8 +44,8 @@ def init(param_test):
     param_test.groundtruth = [
         [0.00047529041677414337, -1.1970542445283172e-05, -1.1970542445283172e-05, -1.1970542445283172e-05, -0.1296642741802682,
          -0.1296642741802682, -0.1296642741802682],
-        [0.008491259664160821, 0.0029020670607778237, 0.0029020670607778237, 0.0029020670607778237, -0.014405996135095476,
-         -0.014405996135095476, -0.014405996135095476],
+        [0.008032332623754357, 0.0037734940916436697, 0.0037734940916436697, 0.0037734940916436697,
+         -0.01502861167728611, -0.01502861167728611, -0.01502861167728611],
```

Comment:
I've changed the ground truth of this test because of the reasoning in https://github.com/neuropoly/spinalcordtoolbox/pull/3270#discussion_r593780155.

Could you double-check the reasoning behind this change, @jcohenadad? (I'd request your review, but this is your PR, so it won't let me :sweat_smile:.)

Reply:
the reasoning makes sense, thank you!